soh/soh/OTRGlobals.cpp

```diff
@@ -1385,10 +1385,6 @@ extern "C" void Graph_StartFrame() {
             break;
         }
 #endif
-        case KbScancode::LUS_KB_F11: {
-            CVarSetInteger(CVAR_SETTING("Fullscreen"), !CVarGetInteger(CVAR_SETTING("Fullscreen"), 0));
-            break;
-        }
```

Comment:
Does this mean we lose F11 fullscreen or does that still work via something internal to LUS

Reply:
The actual behavior of _toggling_ via F11 is on the LUS side and still functional. This was just a secondary hotkey handler for updating the CVar soley for the checkbox (so not needed)

Reply:
Cool, just checking